examples/app-npm-webpack/messages/ru.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ru": {
+		"intro-1": "Используйте Globalize для интернационализиции вашего приложения.",
+		"number-label": "Независимое число",
+		"currency-label": "Независимая валюта",
+		"date-label": "Независимая дата",
+		"relative-time-label": "Независимое относительное время",
+		"message-1": "Пример сообщения со смешанными числами \"{number}\", валютой \"{currency}\", датами \"{date}\", и относительным временем \"{relativeTime}\".",
+		"message-2": [
+			"Пример сообщения с поддержкой множественного числа:",
+			"{count, plural,",
+			"   one {У вас осталась одна задача}",
+			"   two {У вас осталось две задачи}",
+			"   three {У вас осталось три задачи}",
+			"   four {У вас осталось четыре задачи}",
+			"   five {У вас осталось пять задач}",
+			"   few {У вас осталось несколько задач}",
+			"   many {У вас еще много задач}",
+			"   other {У вас еще много задач}",
```

Comment:
I'm not sure how "other" numerals ending could be described like this,  since logic of it should look like this - 
```js
function declOfNum(number, titles) {  
    cases = [2, 0, 1, 1, 1, 2];  
    return titles[ (number%100>4 && number%100<20)? 2 : cases[(number%10<5)?number%10:5] ];  
}
declOfNum(count, ['найдена', 'найдено', 'найдены']);
``` 
so i put `other` as equivalent of `many` assuming that any other number then 1..5 is "a lot".